coursier.bzl

```diff
@@ -43,6 +43,7 @@ load("%s", "aar_import")
 """
 
 _BUILD_PIN = """
+exports_files(['unsorted_deps.json'])
```

Comment:
Is this line required, since it's not referenced outside of the package?

Reply:
Good catch, doesn't seem to be required. Have updated the PR.